cmd/buildah/push.go

```diff
@@ -104,8 +110,26 @@ func pushCmd(c *cli.Context) error {
 		return errors.Wrapf(err, "error building system context")
 	}
 
+	var manifestType string
+	transport := strings.SplitN(args[1], ":", 2)[0]
+	if transport == "dir" {
+		if c.IsSet("format") {
+			switch c.String("format") {
+			case "oci":
+				manifestType = imgspecv1.MediaTypeImageManifest
+			case "v2s1":
+				manifestType = manifest.DockerV2Schema1SignedMediaType
+			case "v2s2":
+				manifestType = manifest.DockerV2Schema2MediaType
```

Comment:
The set of values we recognize here doesn't match the set of values for `--format` that `commit` accepts.  Can add  `docker` as an alias for `v2s2`?

Reply:
added `docker` as an alias for `v2s2`. Fixed